package.json

```diff
@@ -14,10 +14,9 @@
     "lint": "tslint ./lib/**/*.ts",
     "prettier": "prettier lib/**/*.ts examples/**/* --write",
     "lint:fix": "yarn lint -- --fix",
-    "test": "NODE_ENV=development yarn build:examples && rollup --environment NODE_ENV:'development' -c rollup.config.js && yarn build:test",
-    "build": "rollup --environment NODE_ENV:'production' -c rollup.config.js",
+    "test": "NODE_ENV=development yarn build:examples && yarn build && jest",
+    "build": "rollup --environment NODE_ENV:'production' -c rollup.config.js && bundlesize",
```

Comment:
We can add a script called `test:size` rather than running Bundlesize in the `build` script (those have different purposes).